docs/functional-components/static-assets.md

```diff
@@ -0,0 +1,34 @@
+## Static assets
+
+## Recommended packages
+
+* [express.static][]: `express.static` is part of the Express.js package that allows developers to expose static middlewares.
+
+## Guidance
+
+For serving static resources on the application, we recommend using [express.static][] middleware as it has been widely used and tested in production.
+
+When using `express.static()` method, we can serve static resources directly by specifying the folder name where we have stored our static resources.
+
+Documentation for the middleware can be found [here][express.static]
+
+### Caching
+
+The static middleware allows caching of static resources via exposed [caching-headers][].
+
+Origin servers communicate caching instructions via the header `Cache-Control`. The values of `Cache-Control` are called directives. Example directives include: `max-age`, `no-store, no-cache, must-revalidate`, `public, private`, etc.
```

Comment:
Expand directives to particular use cases:

Eg. depending on content, use a particular directive

Reply:
I think you also indicated you'd update to include 

* static content only (ie no need to run JS) -> just use nginx
* static content/with JS -> pretty much what we have, bundle together, rely on CDN to cache the static content effectively
* static content that is uploaded -> separate out